src/pages/Home/ReservesRanges/DateRange/DateRange.tsx

```diff
@@ -46,6 +55,7 @@ const DateRange: React.FC<IDateRange> = ({ startDateProps, endDateProps, recruit
             </MenuItem>
           ))}
         </TextField>
+        </Box>
       </Grid>
```

Comment:
I see that in a mobile view, each of the DataInput and MenuItem is displayed on a separate line.
![Screenshot 2024-02-17 at 22 40 05](https://github.com/CalculatorMiluim/Calculator-Miluim/assets/58932509/1d3f15f2-af9a-4e7c-9a7f-82cfa2ac87f1)

This can cause some confusion when we will add the option to add multiple date ranges
![Screenshot 2024-02-17 at 22 51 28](https://github.com/CalculatorMiluim/Calculator-Miluim/assets/58932509/fc2326ad-a69b-4224-91bb-22faf5c44980)

I think the best solution for mobile support would be to use a single date range input. like we have in the figma:
![Screenshot 2024-02-17 at 22 54 52](https://github.com/CalculatorMiluim/Calculator-Miluim/assets/58932509/e05dddca-dd29-4dcb-9687-678786f6b467)

Reply:
ok, I will include in this ticket the component change,  thanks

Reply:
@EfratGutman 
this kind of component, showing range dates inside the same field, needs to be a custom component since mui does not provide something like this.
Please confirm that this is what we want to do.
Thanks